sky/clouds/nebius.py

```diff
@@ -378,3 +376,10 @@ def validate_region_zone(self, region: Optional[str], zone: Optional[str]):
         return service_catalog.validate_region_zone(region,
                                                     zone,
                                                     clouds='nebius')
+
+    @classmethod
+    def get_user_identities(cls) -> Optional[List[List[str]]]:
+        sdk = nebius.sdk()
+        profile_client = nebius.iam().ProfileServiceClient(sdk)
+        profile = profile_client.get(nebius.iam().GetProfileRequest()).wait()
+        return [[profile.service_account_profile.info.metadata.name]]
```

Comment:
Similar to the GCP handling for different workspace config, should we handle the lru_cache and the workspace switching correctly here as well?

https://github.com/skypilot-org/skypilot/blob/0f8da3bcbe0457f2bcf2c04b856209ea1b251e20/sky/clouds/gcp.py#L999-L1047